types/unl-core/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for unl-core 2.1
+// Type definitions for unl-core 3.0
```

Comment:
@eturan Had to re-push to update the version. Could you please re-approve when you get a chance? :)